azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -641,7 +641,14 @@ private void getRunningFlowsIdsHelper(List<Integer> allIds,
 
   @Override
   public List<ExecutableFlow> getRecentlyFinishedFlows() {
-    return new ArrayList<ExecutableFlow>(recentlyFinished.values());
+    List<ExecutableFlow> flows = new ArrayList<>();
+    try {
+      flows = executorLoader.fetchRecentlyFinishedFlows(
+          RECENTLY_FINISHED_LIFETIME.toMillis());
+    } catch(ExecutorManagerException e) {
+      logger.error(e);
```

Comment:
Changed to error(Object message, Throwable t). I think it's ok to just keep the existing behavior to return an empty list if it fails to fetch recently finished flows. What is the intention of returning a server error?